bucket/fun.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "3.0.8",
+    "description": "A development tool for serverless applications on Alibaba cloud.",
+    "homepage": "https://github.com/alibaba/funcraft",
+    "license": "MIT",
+    "url": "https://gosspublic.alicdn.com/fun/fun-v3.0.8-win.exe.zip",
+    "hash": "72d148c000efb16d8ec8a2355e157b4904e58da1d50dd1ee6187cb70ef516e21",
+    "bin": [
```

Comment:
```suggestion
    "pre_install": "Rename-Item \"$dir\\fun*win.exe\" 'fun.exe'",
    "bin": "fun.exe",
```

Reply:
This doesn't work for me.

```
❯ rename-item ./a*.txt a2.txt
rename-item : Cannot process argument because the value of argument "path" is
 not valid. Change the value of the "path" argument and run the operation aga
in.
At line:1 char:1
+ rename-item ./a*.txt a2.txt
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Rename-Item], PSArgument
   Exception
    + FullyQualifiedErrorId : Argument,Microsoft.PowerShell.Commands.RenameI
   temCommand
```

move-item works.

Reply:
![A](https://i.imgur.com/w4naJHD.png)

Reply:
wow, something went wrong here... Sorry for the spam.
It's strange, rename-item does not work with wildcard for me.

Reply:
But feel free to suggest move-item

![A](https://i.imgur.com/WTrfxdE.png)f

Reply:
I see an insteresting thing. It works on my C disk but not work on my D disk.

![2019-11-06 15_30_37-test](https://user-images.githubusercontent.com/36977733/68277570-60917b80-0067-11ea-871f-4c5f617dffb8.png)